runbotics-orchestrator/src/main/java/com/runbotics/repository/TenantInviteCodeRepository.java

```diff
@@ -15,9 +15,4 @@
  */
 @Repository
 public interface TenantInviteCodeRepository extends JpaRepository<TenantInviteCode, UUID>, JpaSpecificationExecutor<TenantInviteCode> {
```

Comment:
is this empty interface kept to not modify rb-orchestrator anymore or for keeping human readable name?

Reply:
It may look like it is empty, but this interface extends JpaRepository, so with this we are able to access default methods provided by JPA like findAll etc.